lib/src/main/java/com/diffplug/spotless/npm/TsFmtFormatterStep.java

```diff
@@ -57,11 +71,11 @@ public static FormatterStep create(Provisioner provisioner, File buildDir, @Null
 
 		private final File baseDir;
 
-		public State(String stepName, Provisioner provisioner, File buildDir, @Nullable File npm, File baseDir, @Nullable TypedTsFmtConfigFile configFile, @Nullable Map<String, Object> inlineTsFmtSettings) throws IOException {
```

Comment:
Since this is `public`, let's keep it for backward compatibility but add an `@Deprecated` and delegate to the new constructor.